components/browser/menu/src/main/java/mozilla/components/browser/menu/item/BrowserMenuDivider.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.menu.item
+
+import android.view.View
+import mozilla.components.browser.menu.BrowserMenu
+import mozilla.components.browser.menu.BrowserMenuItem
+import mozilla.components.browser.menu.R
+
+/**
+ * A browser menu item to display a horizontal divider.
+ */
+class BrowserMenuDivider : BrowserMenuItem {
```

Comment:
Looking at the other classes I wonder if this should be `BrowserMenuDividerItem`?

Reply:
I was a bit confused with this one because there were more classes that are not adding the item prefix than the ones that have it, 3 vs 2.  I thought they didn't because they already are in the `item` package.

Should I add it too?

![image](https://user-images.githubusercontent.com/773158/51202792-8ded1d00-18cd-11e9-8ad0-794b9b743814.png)

Reply:
Alright. In this case let's leave it as-is. :)

Reply:
Should I rename the others ?

Reply:
Oops that could be an Api break, now we have to live with that